### ISSUE TITLE
Visual Studio 2015 UAP project fails to build

### ISSUE THREAD

**Issue by sondreb** [winjs/winjs#1321]
Not sure if this is the right place to mention, but the Blank App under JavaScript/Windows/Universal that comes with the latest SDKs for Visual Studio 2015, does not work.

""10.0.0.0" is not a supported value for TargetPlatformVersion"

I created a bug here: https://connect.microsoft.com/VisualStudio/feedback/details/1606376

Can anyone reproduce this error? It makes it impossible to build Windows 10 apps with WinJS.

**Comment by seksenov** [comment]
Do you know what the version of Visual Studio 2015 and the SDK that you have installed is?

I've seen this error many times and it's usually a mismatch between the installed SDK that you have and what's been set in the TargetPlatformVersion under the project's properties.

**Comment by seksenov** [comment]
Do you know what version of the SDK you have installed? Also was this a clean install of VS 2015 and the tools or did you have the SDK installed on the past? 

It looks like you are loading a pre RTM project type it should be Windows Universal rather than UAP which leads me to believe there is something wrong with the SDK install.

**Comment by sondreb** [comment]
Installed Windows 10 earlier today, so a clean installation.
Installed Visual Studio 2015 ISO (Enterprise) released 20th of July.
Ran the VSToolsForWindows1E.exe which added the emulator and SDK for Windows 10.

VS Version: 14.0.23107.0 D14REL
.NET Version: 46.00079
Visual Studio Tools for Universal Apps: 14.0.23121.00 D14OOB.

I do have the Node.js Tools (RC) for Visual Studio installed, don't think the "Universal" option was available before running the SDK install mentioned above. Azure 2.7 SDK and Cordova options is also installed.

I do get this for a C# Universal App, so something is not entirely right:

<img width="403" alt="sdk_error" src="https://cloud.githubusercontent.com/assets/309938/8970331/7a90dbee-3649-11e5-9db7-bc13c17433fc.PNG">

**Comment by sondreb** [comment]
Running the vs_enterprise.exe and choosing Modify, gave me this... strange, or what?

Perhaps the SDK tools thing was unable to update that package? Will report back after changing to Update 1:

<img width="283" alt="vs_modify" src="https://cloud.githubusercontent.com/assets/309938/8970585/d142a53e-364a-11e5-8897-c0abe922c80b.PNG">

**Comment by seksenov** [comment]
Looks like the 10.0.10240 SDK wasn't installed. Building and deployment of apps should work after the update. Let me know how it goes and I will dig in if you're still having issues.

**Comment by sondreb** [comment]
First I changed to the Update 1 option, which broke something with this error when creating JavaScript Universal project:

<img width="345" alt="after_update1" src="https://cloud.githubusercontent.com/assets/309938/8972305/11f160f8-3659-11e5-8cb7-3e353e593d16.PNG">

I had to uninstall the SDK (VSToolsForWindows1E.exe) and install again, then both C# and JavaScript projects works fine to create and build.

Though debugging the "Universal Windows" JavaScript app fails with this error:

<img width="344" alt="debugging" src="https://cloud.githubusercontent.com/assets/309938/8972269/bf2dc2bc-3658-11e5-8cee-67f5d3230c10.PNG">

Properties dialog on the project shows this error, and does not open anything:

<img width="301" alt="properties_error" src="https://cloud.githubusercontent.com/assets/309938/8972278/df8d1f3a-3658-11e5-88ee-fee234b651b2.PNG">

Both of which I'm sure is related to the first error message with ToolsPackage not loading.

From installed addons:

<img width="537" alt="cordova_tools" src="https://cloud.githubusercontent.com/assets/309938/8972348/8e778e22-3659-11e5-8542-4b500516a459.PNG">

Running VS setup with modify, it now says checkbox on both the Replace version and the Update 1 version of the Apache Cordova option. Unchecked both and removing... then I will try to add them again tomorrow.

**Comment by sondreb** [comment]
Uninstalling the Cordova tools had no affect, other than removing that error dialog. I looked in the log and it appears the same add-in was registered twice, yet still unrelated to the Universal Windows app type... so no good. Might do a reset of the OS tomorrow, but very annoying as I've spent a bunch of hours setting everything up from a clean machine this morning.

**Comment by sondreb** [comment]
Reset Windows 10 (keep files), then running the new Visual Studio 2015 Enterprise with Tools web installer fixed the problem, now both C# and JavaScript based Universal Apps builds and debugs.

I do think this issue was related somehow to one or multiple packages for Visual Studio 2015 which have been updated since RTM 20th July (see the Update 1 for Cordova). I updated the issue on Connect with a comment as well.
